### PR TITLE
fixed blindness problem

### PR DESCRIPTION
**Pull request by giordanoscerra**
yes. fixed a 3/16 chance problem. yup.

### CODE REVIEW

File: utils/agent.py [hunk 1]
```diff
@@ -175,8 +179,8 @@ def process_inventory(self, game_map:Map, interesting_items:list = ['saddle', 'c
 
     def act(self, level:Map, show_steps:bool=True, graphic:bool = False, delay:float = 0.1):
         self.current_subtask = self.kb.query_for_action() # returns subtask to execute
-        #print("\n\n UHM. the voices in my head are telling me to", self.current_subtask, "!!!!!!!!!!!!!!")
-        #time.sleep(0.5)
+        print("\n\n UHM. the voices in my head are telling me to", self.current_subtask, "!!!!!!!!!!!!!!")
+        time.sleep(0.5)
```
Comment (Chomp97) [inline utils/agent.py:183]: !?!?!?!?!

Comment (M4rinz) [inline utils/agent.py:183]: Remoooooove! >:(

**Review by M4rinz — COMMENTED** [review]
11/10 would eat again

File: utils/agent.py (hunk 1 above)
Comment (giordanoscerra) [reply utils/agent.py:183]: forza roma

**Pull request merged**